docker-compose.prod.yml

```diff
@@ -5,3 +5,4 @@ services:
         env_file:
             - prod.env
         command: [ "bot" ]
+        restart: always
```

Comment:
I have seen `unless-stopped` also used but not sure if it makes a difference

Reply:
https://docs.docker.com/config/containers/start-containers-automatically/
`always` just seemed simpler